code/modules/surgery/gastrectomy.dm

```diff
@@ -0,0 +1,48 @@
+/datum/surgery/gastrectomy
+	name = "Gastrectomy"
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_real_bodypart = TRUE
+	steps = list(/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/incise,
+		/datum/surgery_step/gastrectomy,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/close
+		)
+
+/datum/surgery/gastrectomy/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/liver/L = target.getorganslot(ORGAN_SLOT_STOMACH)
+	if(L?.damage > 50 && !(L.organ_flags & ORGAN_FAILING))
+		return TRUE
```

Comment:
You mean `/obj/item/organ/stomach`, right?

Reply:
Do you ever begin writing a PR at 4am and type what you think about consistency through liver damage and fuck up along your lines? Yyyeah...my bad